package.json

```
@@ -11,7 +11,7 @@
     "build:rspack": "pnpm --filter \"example-*\" build",
     "test:rspack": "pnpm --filter \"example-*\" test",
     "build:rspress": "pnpm --filter \"rspress-*\" build",
-    "prepare": "husky 2>/dev/null || true",
+    "prepare": "husky 2>/dev/null || exit 0",
```

Let's use the pure husky command: https://typicode.github.io/husky/get-started.html#husky-init-recommended

```suggestion
    "prepare": "husky",
```